main.py

```diff
@@ -0,0 +1,91 @@
+number_1 = 3
+number_2 = 4
+suma1 = number_1 + number_2
+print(suma1)
+
+roznica1 = number_1 - number_2
+print(roznica1)
+
+mnozenie1 = number_1 * number_2
+print(mnozenie1)
+
+dzielenie1 = number_1 / number_2
+print(dzielenie1)
+
+
+
+number_3 = 0.1
+number_4 = 8.2
+
+suma2 = number_3 + number_4
+print(suma2)
+
+roznica2 = number_3 - number_4
+print(roznica2)
+
+mnozenie2 = number_3 * number_4
+print(mnozenie2)
+
+dzielenie2 = number_3 / number_4
+print(dzielenie2)
+
+
+
+liczba1 = 10
+
+liczba2 = 0
+
+dzielenieprzezzero = liczba1 / liczba2
+
+print(dzielenieprzezzero)
+
+# wykonałem dzielenie przez zero, takie działanie zawsze daje wynik 0
+
+
+
+
+number5 = 2
+
+number6 = 5
+
+potegowanie = number5 ** number6
+
+print(potegowanie)
+
+
+
+jakasliczba = 0
+
+if jakasliczba >= 0:
+print("Jestem")
+
+if jakasliczba = 5:
+print("Jestem")
+
+# wykonałem warunek który nie spełniał wymogów
+
+jakasliczba = 0
+
+if jakasliczba = 5:
+print("Liczba jest równa zeru")
+
+else:
+print("Liczba jest równa zeru")
+
+
+
+
+
+
+
```

Comment:
Nie zostawiaj tyle pustych lini, to wprowadza bałagan. Staraj się mieć jedna pustą linie na koncu pliku